src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -492,7 +492,7 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
     expect(screen.baseElement).not.to.contain.text('Cancel appointment');
 
     expect(screen.baseElement).to.contain.text(
-      'Contact this provider if you need to reschedule or cancel your appointment.',
+      'Contact this facility if you need to reschedule or cancel your',
```

Comment:
COVID appointments had a cancel alert telling them to contact a facility, with slightly different wording than the video appointment one, so I switched it to use the same one as video appointments.